enterprise/frontend/src/metabase-enterprise/google_drive/utils.tsx

```diff
@@ -40,13 +47,9 @@ export const getStatus = ({
     .otherwise(() => "not-connected");
 
 export const getErrorMessage = (
-  payload:
-    | unknown
-    | string
-    | { data: { message: string } | { error_message: string } | string }
-    | { message: string }
-    | { error_message: string },
-  fallback: string = t`Something went wrong`,
+  payload: ErrorPayload,
```

Comment:
Any chance we could improve the `getErrorMessage` function in `metabase/api/utils/settings` to do what this does? Both functions are nearly identical. There are probably other places where we have bespoke functions for this too

Reply:
good call, I'll make a separate task to unify these